containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -689,6 +694,14 @@ TEST(TEST_CATEGORY, create_mirror_no_init_dynamicview_view_ctor) {
                                Kokkos::DefaultExecutionSpace{}),
             host_view);
       },
+      [&](BeginFenceEvent event) {
+        if (event.descriptor().find("DynamicView::resize_serial: Fence after "
+                                    "copying chunks to the device") !=
+            std::string::npos)
+          return MatchDiagnostic{false};
```

Comment:
We should probably introduce a version that accepts an execution space instance for the `deep_copy` later.